classes/submission/PKPSubmissionMetadataFormImplementation.inc.php

```diff
@@ -105,6 +105,11 @@ function initData($submission) {
 			// include all submission metadata fields for submissions
 			$this->_parentForm->setData('submissionSettings', array('all' => true));
 			$this->_parentForm->setData('abstractsRequired', $this->_getAbstractsRequired($submission));
+			
+			$submissionDao = Application::getSubmissionDAO();
+			$submissionSettingsRevisions = $submissionDao->getSubmissionRevisions($submission->getId(), true, true, 'DESC'); 
```

Comment:
You can use the `SORT_DIRECTION_...` constants defined in `DAO.inc.php` instead of ASC/DESC strings.